src/views/Player.vue

```diff
@@ -255,9 +264,6 @@ export default {
     },
 
     submitAnswer(plioQuestion, answer) {
-      // start playing whenever the user submits an answer
-      this.player.play()
-
```

Comment:
moved this to the end.
Why? 
Because whenever the answer was submitted, the first thing that happened is the player started playing. The rest of the process was happening later. The problem with player started playing instantly was that the currentTime starts getting updated, and if another question is very very close to the current one, it again brings up those bugs.

I'm not sure how much optimization will this bring, but seems logical.